service/telemetry/logger.go

```diff
@@ -39,6 +39,20 @@ func newLogger(set Settings, cfg Config) (*zap.Logger, log.LoggerProvider, error
 		return nil, nil, err
 	}
 
+	// The resouce attributes are added to the logger sending the logs to the OTLP endpoint
+	// but not to the one handling the STDOUT from the collector. Here, we add them
+	// to the logger handling the STDOUT from the collector.
```

Comment:
Suggestion on how to explain the use of WrapCore more clearly:
```suggestion
	// The attributes in cfg.Resource are added as resource attributes for logs exported through the LoggerProvider instantiated below.
	// To make sure they are also exposed in logs written to stdout, we add them as fields to the Zap core created above using WrapCore.
	// We do NOT add them to the logger using With, because that would apply to all logs, even ones exported through the OTel core, meaning that the attributes would be exported twice.
```